res/values-ru

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2017-2019 The PixelDust Project
```

Comment:
Copyright header?

Reply:
I copied from the original file this is fine